vxquery-xtest/src/main/java/org/apache/vxquery/xtest/XTest.java

```diff
@@ -16,15 +16,15 @@
  */
 package org.apache.vxquery.xtest;
 
+import org.mortbay.jetty.Server;
```

Comment:
Do not add unnecessary imports. Please our code formatting guide: http://vxquery.apache.org/development_eclipse_setup.html